File: src/main/java/net/guizhanss/ultimategenerators2/utils/RandomUtils.java
package net.guizhanss.ultimategenerators2.utils;

import java.util.concurrent.ThreadLocalRandom;

import lombok.experimental.UtilityClass;

@UtilityClass
public final class RandomUtils {
    public static boolean nextBoolean() {
        return ThreadLocalRandom.current().nextBoolean();
    }

    public static float nextFloat(float origin, float bound) {
        return ThreadLocalRandom.current().nextFloat(origin, bound);
    }
}

File: src/main/kotlin/net/guizhanss/ultimategenerators2/utils/RandomUtils.kt
package net.guizhanss.ultimategenerators2.utils

import java.util.concurrent.ThreadLocalRandom

object RandomUtils {
    fun nextBoolean(): Boolean {
        return ThreadLocalRandom.current().nextBoolean()
    }

    fun nextFloat(origin: Float, bound: Float): Float {
        return ThreadLocalRandom.current().nextFloat(origin, bound)
    }
}
